svc-task/main.go

```diff
@@ -43,10 +44,11 @@ func main() {
 
 	config.CollectCLArgs()
 
-	if err := dc.SetConfiguration(config.Data.MessageQueueConfigFilePath); err != nil {
-		log.Fatal("error while trying to set messagebus configuration: " + err.Error())
+	if strings.EqualFold(config.Data.MessageBusConf.MessageBusType, "kafka") {
+		if err := dc.SetConfiguration(config.Data.MessageBusConf.MessageQueueConfigFilePath); err != nil {
+			log.Fatal("error while trying to set messagebus configuration: " + err.Error())
+		}
```

Comment:
I can see only kafka condition, similarly add code for redisstreams

Reply:
we are setting configuration for Kafka, there is no such config method in this PR